### DISCUSSION TITLE
皮肤文件夹损坏，皮肤显示不出

### DISCUSSION THREAD

**Discussion by ZaoCeng** [Meloong-Git/PCL#3375] [提问] [answered]
为什么我的pcl显示不出皮肤？问别人的时候他们发我的链接我打开是我皮肤的展开图，请问要怎么解决？

**Comment by MoYuan-CN** [comment]
皮肤文件就应当是展开图
在下载了图片之后，打开 PCL 启动器 -> 设置 -> 离线皮肤 -> 自定义 然后选你下载的文件就行

**Reply by ZaoCeng** [reply]
如果我用的是官方的账号登录，然后进游戏皮肤没显示要怎么办？

**Reply by MoYuan-CN** [reply]
登陆后主页不显示皮肤问题：见下

**Comment by allMagicNB** [comment]
#3289

**Comment by WForst-Breeze** [comment, marked as answer]
请尝试使用 VPN 后再试
